WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -942,6 +942,18 @@ public boolean isNewPost() {
         return mIsNewPost;
     }
 
+    /*
+     * returns true if the user has permission to publish the post - assumed to be true for
+     * dot.org sites because we can't retrieve their capabilities
+     */
+    private boolean canPublishPost() {
```

Comment:
May I suggest calling this `userCanPublishPosts`? There are lots of nuances to whether a post can be published (e.g. we have [`isPublishable` checks](https://github.com/wordpress-mobile/WordPress-Android/blob/15c3cba2e51a79a931d543401c91a8422246f509/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L1083) in several places), and I think we should keep things as specific as possible.